packages/grafana-schema/src/schema/mudball.gen.ts

```diff
@@ -585,3 +585,57 @@ export type TimeZoneBrowser = 'browser';
 export type TimeZone = (TimeZoneUtc | TimeZoneBrowser | string);
 
 export const defaultTimeZone: TimeZone = 'browser';
+
+export interface MapLayerOptions {
+  /**
+   * Custom options depending on the type
+   */
+  config?: unknown;
```

Comment:
```suggestion
  config?: T;
```
any way to make this generic?

Reply:
This is generated file from cue, which doesn't support generics. Current approach is to add a `veneer` layer that would add generics, which [done here](https://github.com/grafana/grafana/pull/61636/files#diff-6e1bcc541cb23781472d359a26f6093e1633cf070bee2bfa18d547a4fe122919R5). So basically these types are the base, which are then extended and reexported in `mudball.types.ts`.

Reply:
nice